templates/partners/channel-and-reseller.html

```diff
@@ -0,0 +1,131 @@
+      </div>
+      <div class="col-4 u-hide--small u-align--center u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/80472358-Canonical-reseller-icon.svg" alt="">
+      </div>
+    </div>
+  </div>
+</section>
+<style>
+  .p-strip--suru-top {
+    background-image:
+      linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
+      linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.34) 50%, rgba(74, 24, 60, 0.34) 100%),
+      linear-gradient(to top left, #f7f7f7 0%, #f7f7f7 49%, transparent 50%),
+      linear-gradient(230deg, #E95420 0%, #772953 33%, #2C001E 72%);
+    background-position: left top, right top, right bottom;
+    background-repeat: no-repeat;
+    background-size: 70% 80%, 70% 100%, 110% 20%, 100% 100%;
+    padding-bottom: 6rem;
+    padding-top: 3rem;
+  }
+
+  @media (max-width: 620px) {
+    .p-strip--suru-top {
+      background-image:
+        linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.4) 50%, rgba(74, 24, 60, 0.4) 100%),
+        linear-gradient(to top left, #f7f7f7 0%, #f7f7f7 49%, transparent 50%),
+        linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
+        linear-gradient(120deg, #2C001E 4%, #4C193E 100%);
+      background-position: right top, bottom right, top left, top left, center right;
+      background-repeat: no-repeat;
+      background-size: 100% 100%, 105% 25%, 70% 80%, 100% 100%;
+    }
+  }
+</style>
+<section class="p-strip--light is-shallow" style="margin-top: -1px;">
+  <div class="row">
+    <div class="col-8">
+      <p>Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
+    </div>
+    <div class="col-4">
+      <a href="/partners/become-a-partner" class="p-button--positive">Get in touch</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip is-deep">
+  <div class="u-fixed-width u-align--center">
+    <h4 class="p-muted-heading u-no-max-width">Meet some of our channel partners</h4>
+  </div>
+  <div class="u-fixed-width">
+    <ul class="p-inline-images u-no-margin--bottom">
+      <li class="p-inline-images__item">
+        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/cb3fcdfb-lenovo.svg" alt="Lenovo">
+      </li>
+      <li class="p-inline-images__item" style="display: inline; padding-top: 0.5rem;">
+        <img class="p-inline-images__logo" src="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg" alt="Dell">
```

Comment:
The dell logo is displaying weirdly for me:
![image](https://user-images.githubusercontent.com/118614/79958402-c7bab500-847a-11ea-97f8-97754fa6e4bd.png)

I know that I've mentioned this on other pages but these logo clouds shouldn't be hard-coded. We want non-webteam people to be able to change them in the CMS

Reply:
I will need to update all the partners pages to use the CMS data. I have a [separate issue](https://github.com/canonical-web-and-design/canonical.com/issues/214) for that.

Reply:
Ah, no problem